modules/combined/doc/content/modules/combined/examples/current_heating_of_wire.md

```diff
@@ -0,0 +1,87 @@
+# Coupling electromagnetics and heat transfer modules for wire heating
+
+The purpose of this example is to present how the [electromagnetics](modules/electromagnetics/index.md)
+and [heat transfer](modules/heat_transfer/index.md) modules can be coupled together to simulate the heating of
+materials through electromagnetic effects. In particular, the example presented here involves simulating the
+heating profile of a copper wire when suppled with the fusing current.
+
+## Problem Description
+
+The problem of interest involves suppling large amounts of DC current to a copper wire, such that the temperature
+of the wire reaches the melting point. To model this be behavior, the follow equations are utilized:
+
+\begin{equation}
+  \nabla \times \nabla \times \vec{A} + j \mu \omega \left( \sigma \vec{A} \right) = \vec{J}_{source}
+\end{equation}
+\begin{equation}
+  \rho C_{p} \frac{\partial T}{\partial t} - \nabla \cdot k \nabla T = Q
+\end{equation}
+
+Where:
+
+- $\vec{A}$ is the magnetic vector potential,
+- $j$ is the sqrt(-1),
+- $\mu$ is the permeability of free space,
+- $\omega$ is the angular frequency of the system (60 Hz for this example),
+- $\sigma$ is the electric conductivity of the wire,
+- $\vec{J}_{source}$ is the supplied DC current density,
+- $\rho$ is the density of copper,
+- $C_{p}$ is the heat capacity of copper,
+- $T$ is the temperature,
+- $k$ is the thermal conductivity of the wire, and
+- $Q$ is the Joule heating.
+
+The Joule heating, $Q$, is defined by the magnitude of the electric field, $E$, such that:
+\begin{equation}
+  Q = \frac{1}{2} \sigma \lvert \vec{E} \rvert ^{2}
+\end{equation}
+
+Instead of solving for the electric field directly, this example used the magnetic vector potential
+because the magnetic vector potential formulation allows for the directly implementation of a
+suppled DC current density. This implementation of a suppled DC current density can be better displayed by looking at
+the relationship between the electric field and the magnetic vector potential:
+
+\begin{equation}
+  \vec{E} = -j \omega \vec{A} - \nabla \left( V \right)
```

Comment:
```suggestion
  \vec{E} = -j \omega \vec{A} - \nabla V
```

I don't think it's necessary to have the parentheses here, especially since we're not taking the gradient of a multi-term expression.

Reply:
Please correct this elsewhere in this document as well.

Reply:
Latest commit corrected this.